### PR TITLE
{lib}[GCCcore/9.3.0] CLISP v2.49, DB v18.1.32, libffcall v2.2, ...

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)
updated from #10071

### CODE REVIEW

**Comment by smoors** [comment]
Test report by @smoors
**SUCCESS**
Build succeeded for 8 out of 8 (5 easyconfigs in this PR)
node314.hydra.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 2.7.5
See https://gist.github.com/350fe961c5d5e0d1abcda8f3788af176 for a full test report.

File: easybuild/easyconfigs/c/CLISP/CLISP-2.49-GCCcore-9.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    ('PCRE', '8.43'),
+    ('zlib', '1.2.11'),
+]
+
+configopts = 'build'
+prebuildopts = """
+    # disable "streams" test due to problems with https://
+    # https://sourceforge.net/p/clisp/mailman/message/36224219/
+    sed -e 's/"streams"/"streamslong"/' -i.eb tests/tests.lisp
+    cd build
```
Comment (boegel) [inline easybuild/easyconfigs/c/CLISP/CLISP-2.49-GCCcore-9.3.0.eb]: @smoors I would like to see this differently style-wise, to confirm with existing easyconfigs:

```python
# disable "streams" test due to problems with https://;
# see https://sourceforge.net/p/clisp/mailman/message/36224219/
prebuildopts = "sed -e 's/"streams"/"streamslong"/' -i.eb tests/tests.lisp && cd build && "

**Comment by zao** [comment]
Test report by @zao
**FAILED**
Build succeeded for 3 out of 5 (5 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/5c69bdfd70a06edc7055a1baf2020506 for a full test report.

**Comment by zao** [comment]
Test report by @zao
**SUCCESS**
Build succeeded for 2 out of 2 (5 easyconfigs in this PR)
freja - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8700K CPU @ 3.70GHz, Python 2.7.17
See https://gist.github.com/e389efcaebc8157f24b0b6295480897d for a full test report.

**Comment by akesandgren** [comment]
Test report by @akesandgren
**FAILED**
Build succeeded for 3 out of 5 (5 easyconfigs in this PR)
skalman.hpc2n.umu.se - Linux ubuntu 18.04, Intel(R) Core(TM) i7-8565U CPU @ 1.80GHz, Python 2.7.17
See https://gist.github.com/6c25c4de94b185cb70137413bad202a5 for a full test report.

**Comment by zao** [comment]
A note that the DB download need to be separately obtained would've been nice in the PR itself. One could read the easyconfig to find it out of course, but it appears 2/3 didn't :D

**Comment by smoors** [comment]
@zao I agree with you!
this PR was originally from @JackPerdue  (see #10078 ).
as it happens we already had DB-18.1.32 installed, that's why I did not notice.

**Comment by lexming** [comment]
Test report by @lexming
**FAILED**
Build succeeded for 3 out of 5 (5 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/2e318d5aa52b166cbe0cae77e3847a7f for a full test report.

**Comment by lexming** [comment]
Test report by @lexming
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in this PR)
login2.cerberus.os - Linux centos linux 7.7.1908, Intel(R) Xeon(R) Gold 6126 CPU @ 2.60GHz, Python 2.7.5
See https://gist.github.com/feb5198f42d482eb6c844414cf1ccbec for a full test report.

**Comment by akesandgren** [comment]
Going in, thanks @smoors!

**Pull request merged**